tag/example_test.go

```diff
@@ -19,7 +19,6 @@ import (
 	"log"
 
 	"github.com/census-instrumentation/opencensus-go/tag"
-	"github.com/rakyll/census/tags"
```

Comment:
Like this 👍

Reply:
I will add some linting to avoid this happening again.